frontend/class-twitter.php

```diff
@@ -19,6 +19,11 @@ class WPSEO_Twitter {
 	/**
 	 * @var array Images
 	 */
+	public $images = array();
```

Comment:
I should make this private, because the scope where it's used is also private. Please correct me if I'm wrong.